src/demo/pages/DetailsListPage/examples/DetailsList.Advanced.Example.tsx

```diff
@@ -393,6 +395,10 @@ export class DetailsListAdvancedExample extends React.Component<any, IDetailsLis
     console.log('Item invoked', item, index);
   }
 
+  private _onItemSelected(isSelected: boolean) {
+    console.log('item', isSelected);
```

Comment:
Would be best to output the index to confirm the corresponding item is selected.